composer.json

```diff
@@ -43,7 +43,8 @@
         "burgov/key-value-form-bundle": "to edit assoc multivalue properties. require version 1.0.*"
     },
     "conflict": {
-        "phpcr/phpcr-shell": "<1.0.0-beta1"
+        "phpcr/phpcr-shell": "<1.0.0-beta1",
+        "symfony/framework-bundle": "<2.3.27|>=2.6.0,<2.6.6"
```

Comment:
hm, actually are we saying here that 2.4 and 2.5 are ok, and 2.6.7 and higher 2.6 as well? i guess we are, but do we want to say that? we don't test 2.4, 2.5 or 2.6